Updates pages in the CMS documentation for packages across Umbraco versions. (#6251)
umbraco/UmbracoDocs: 10/umbraco-cms/extending/packages/packages-on-umbraco-cloud.md

2. If you need the content of the files to be different on the different environments you will need to use environment specific [config transforms](https://docs.umbraco.com/umbraco-cloud/set-up/config-transforms).

## Value connectors

A value connector is an extension to Umbraco Deploy that allows you to transform data when you deploy content of any kind between environments. When it comes to packages, one reason you need to consider these if you are supporting deploying content properties that rely on integer IDs. Content and other Umbraco data has two identifiers - an integer and a GUID. The GUID is consistent between environments but the integer ID is not. As such, if transferring content between environments and relying on integer IDs, you'll need to include a value connector to transform the value.

> They also manage dependencies for property data. That means that if you save for example an ID of an image in your property editor, then you can make sure that not only the ID but the actual image is transferred as well.

Reviewer: ```suggestion
They also manage dependencies for property data. If you save an ID of an image in your property editor, you can make sure the ID and the image are transferred as well.
```
